frontend/src/util/preload.ts

```diff
@@ -27,8 +27,11 @@ import { H } from 'highlight.run'
 import moment from 'moment'
 import { useEffect, useRef } from 'react'
 
-const CONCURRENT_PRELOADS = 1
-const PREVIOUS_ERROR_OBJECTS_TO_FETCH = 2
+const CONCURRENT_SESSION_PRELOADS = 1
+const CONCURRENT_ERROR_PRELOADS = 10
+// disable this for now as loading session data for each error instance requires a lot of preloading
+const SHOULD_PRELOAD_ERROR_INSTANCE_SESSION = false
```

Comment:
Is there a reason we can't just remove the code that prefetches a session for an error instance instead?

Reply:
was thinking to leave this here in case we want to enable it in the future, but it's in git anyways so we can always bring this up. thanks for the suggestion; cleaning this up